requirements.txt

```diff
@@ -1,7 +1,7 @@
 pygithub==1.52
 dnspython==2.0.0
 pyyaml==5.3.1
-git+git://github.com/protontypes/coinbase-python@master
+coinbase
```

Comment:
Is there a way to keep the git dependency? The fork has a fix for paging of requests implemented - so we need it until we have an alternative coinbase library.